src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,141 @@
+package duke;
+
+import duke.commands.AddCommand;
+import duke.commands.Command;
+import duke.commands.DeleteCommand;
+import duke.commands.FindCommand;
+import duke.commands.ListCommand;
+import duke.commands.MarkCommand;
+import duke.commands.QuitCommand;
+import duke.commands.UnmarkCommand;
```

Comment:
If all the files in the commands is going to be imported, maybe you could just use
`import duke.commands.*;`
(but I am not sure whether this violates the coding standard but I feel it is more clean)